recipe/meta.yaml

```diff
@@ -40,7 +40,7 @@ source:
     folder: testing
 
 build:
-  number: 54
+  number: 55
```

Comment:
This is the last build for 15.x, and also the last build where we build `pyarrow` on this feedstock...

Fare thee well! 👋

Reply:
https://github.com/conda-forge/arrow-cpp-feedstock/commit/b271605ed43898237f33978d1e6e0ed2292ce31c